usort/types.py

```diff
@@ -46,6 +46,16 @@ class ImportItemComments:
     inline: List[str] = field(factory=list)
     following: List[str] = field(factory=list)
 
+    def __add__(self, other: "ImportItemComments") -> "ImportItemComments":
+        if not isinstance(other, ImportItemComments):  # pragma: nocover
+            return NotImplemented
+
+        return ImportItemComments(
+            before=list(self.before) + other.before,
```

Comment:
What do you think about `[*self.before, *other.before]` -- the `list(...)` had me wondering if we were distrustful of it being a list, or if rhs and lhs are different types somehow.